src/wallet/wallet.ts

```diff
@@ -464,14 +545,12 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
    * @memberof HathorWalletServiceWallet
    * @inner
    */
-  signMessage(seed: string, timestamp: number): string {
-    const xpriv = walletUtils.getXPrivKeyFromSeed(seed, {passphrase: this.passphrase, networkName: this.network.name});
-    const derivedPrivKey = walletUtils.deriveXpriv(xpriv, '0\'');
-    const address = derivedPrivKey.publicKey.toAddress(this.network.getNetwork()).toString();
-
+  signMessage(xpriv: bitcore.HDPrivateKey, timestamp: number): string {
+    // const xpriv = walletUtils.getXPrivKeyFromSeed(seed, {passphrase: this.passphrase, networkName: this.network.name});
```

Comment:
Can we remove this?

Reply:
Done in 3ee21d0a8c4a456520383a10647758af308ad7e7